src/gt4sd/algorithms/conditional_generation/regression_transformer/implementation.py

```diff
@@ -90,9 +90,7 @@ def __init__(
             raise OSError(
                 f"algorithm_version {resources_path.split('/')[-1]} does not exist."
             )
-        self.tokenizer = InferenceBertTokenizer.from_pretrained(
-            resources_path, pad_even=False
-        )
```

Comment:
This was probably changed by mistake, i think we should keep the `pad_even=False` here